### ISSUE TITLE
BUG - Slide up without finger on screen >> NavBar does not appear

### ISSUE THREAD

**Issue by StefaniOSApps** [andreamazz/AMScrollingNavbar#330]
**Describe the bug**
If you scroll up without having your finger on the screen, the navbar is not displayed when you arrive at the top.

**To Reproduce**
1. Open the Demo.
2. Scroll up  without finger like in the video.
3. NavBar does not appear

**Expected behavior**
NavBar should at appear when you arrive at the top (or before)

**Video (ZIP)**
[ScreenRecording_10-12-2018 00-53-41.MP4.zip](https://github.com/andreamazz/AMScrollingNavbar/files/2470982/ScreenRecording_10-12-2018.00-53-41.MP4.zip)

On the 3rd try, it is recognizable that the NavBar does not appear.

**Comment by iniko83** [comment]
It's not bug.
When you release your finger - controller just call checkForPartialScroll(), and navigationBar move up.

**Comment by StefaniOSApps** [comment]
Thanks for the very fast answer :-). I have now developed the previous framework and now use: https://github.com/StefaniOSApps/TLYShyNavBar

**Comment by iniko1983x** [comment]
Yes, I also failed to found Rx extension for scrolling navigationBar & seems also will tuning this solution by self :(

**Comment by StefaniOSApps** [comment]
You are welcome to watch my project. It has a very good demo. Currently it is still faulty on WKWebView. 

> Thanks for the very fast answer :-). I have now developed the previous framework and now use: https://github.com/StefaniOSApps/TLYShyNavBar
